metricbeat/docs/modules/kubernetes.asciidoc

```diff
@@ -85,6 +85,14 @@ metricbeat.modules:
   metricsets:
     - apiserver
   hosts: ["https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}"]
+
+# Kubernetes proxy server
+- module: kubernetes
+  enabled: true
+  metricsets:
+    - kubeproxy
+  hosts: ["localhost:10249"]
```

Comment:
would this work out of the box? it may be worth adding it to https://github.com/elastic/beats/blob/master/deploy/kubernetes/metricbeat/metricbeat-daemonset.yaml

Reply:
It should work if kubeproxy is running locally. If not, the daemonset should do it (updated)